stdlib/runtime/equal.gr

```diff
@@ -1,13 +1,13 @@
 /* grainc-flags --no-pervasives */
 module Equal
 
-include "runtime/unsafe/memory"
-include "runtime/unsafe/wasmi32" as WasmI32
+from "runtime/unsafe/memory" include Memory
+from "runtime/unsafe/wasmi32" include WasmI32
 use WasmI32.{ (==), (!=), (&), (^), (+), (-), (*), (<), remS as (%), (<<) }
-include "runtime/unsafe/wasmi64" as WasmI64
-include "runtime/unsafe/wasmf32" as WasmF32
-include "runtime/unsafe/tags" as Tags
-include "runtime/numbers" as Numbers
+from "runtime/unsafe/wasmi64" include WasmI64
+from "runtime/unsafe/wasmf32" include WasmF32
+from "runtime/unsafe/tags" include Tags
+from "runtime/numbers" include Numbers as Numbers
```

Comment:
Duplicated "Numbers". Do we want the formatter to pun this?

Reply:
It looks like we **only** pun record fields, so we'll avoid punning this right now. @ospencer is going to open an issue to format away all duplicated aliases.